src/pudl/metadata/resources/ferc1.py

```diff
@@ -423,7 +423,9 @@
         "field_namespace": "ferc1",
     },
     "core_pudl__assn_ferc1_pudl_plants": {
-        "description": "FERC 1 Plants and their associated manually assigned PUDL Plant IDs",
+        "description": {
+            "additional_summary_text": "FERC 1 plants and their manually assigned PUDL plant IDs.",
```

Comment:
just extra context totally not blocking
```suggestion
            "additional_summary_text": "FERC 1 plants and their manually assigned PUDL plant IDs.",
            "additional_details_text": "FERC does not assign IDs to plants, so each FERC 1 plant is identified by a ``utility_id_ferc1`` and a ``plant_name_ferc1``."
```